src/logic/otp2/controller.ts

```diff
@@ -721,7 +751,9 @@ export async function searchNonTransit(
                 },
             }
 
-            const [result] = await getTripPatterns(getTripPatternsParams)
+            const [result] = await getAndVerifyTripPatterns(
```

Comment:
Her bør me heller ikkje "verify", for me bryr oss ikkje om RoutingError (grunnen til at me fekk tom liste). Me vil berre returnere eventuelt resultat.

Reply:
Juppjuppjupp